microsoft/testsuites/mshv/cloud_hypervisor_tool.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.executable import Tool
+from lisa.util.process import Process
+
+
+class CloudHypervisor(Tool):
+    @property
+    def command(self) -> str:
+        return "cloud-hypervisor"
+
+    @property
+    def can_install(self) -> bool:
+        # cloud-hypervisor is already installed in MSHV dom0 image.
```

Comment:
We can use like this so far, but in future, the installation should be in this tool. If it's not detected, the install method will be called to install it.

Reply:
cloud-hypervisor is always pre-installed in the MSHV root image so for now I have skipped the installation. If in the future this is made into a common tool, then we would need to implement the installation.